### PR TITLE
🚧 Add snooze logic to rating view

### LINKED ISSUE #2732: Rate us snooze is not working as expected
## Context

When the _rate us_ view is visible and the user doesn't interact with it, it should be presented again after 24 hs.

Build/Branch: develop
Phone OS: iOS

## Expected behavior

After 24 hs, the _rate us_ view should appear again

## Suggestions

Check the logic in MainViewModel, starting in line 280.

### PR DESCRIPTION
**Pull request by sofokli1**
Closes #2732

Just adds some logic to show the view again after 24 hours if the user did not interact the first time it showed up.

🦑 whenever

### CODE REVIEW

File: Toggl.Foundation.MvvmCross/ViewModels/MainViewModel.cs [hunk 1]
```diff
@@ -281,10 +281,27 @@ private void presentRatingViewIfNeeded(bool shouldBevisible)
         {
             if (!shouldBevisible) return;
 
-            if (onboardingStorage.RatingViewOutcome().HasValue) return;
+            var wasShownMoreThanOneTimes = onboardingStorage.NumberOfTimesRatingViewWasShown() > 1;
+            if (wasShownMoreThanOneTimes) return;
```
Comment (khoi) [inline Toggl.Foundation.MvvmCross/ViewModels/MainViewModel.cs]: do we need the extra variable, I feel like
`if (onboardingStorage.NumberOfTimesRatingViewWasShown() > 1)` will suffice

Comment (amulware) [reply Toggl.Foundation.MvvmCross/ViewModels/MainViewModel.cs]: I see no harm in added clarity, but it's up to you guys.

Comment (kornelijepetak) [reply Toggl.Foundation.MvvmCross/ViewModels/MainViewModel.cs]: That variable is used again down there so it has to stay.
However, it should be `OneTime` and not `OneTimes`

Comment (khoi) [reply Toggl.Foundation.MvvmCross/ViewModels/MainViewModel.cs]: or maybe `Once`?

**Pull request merged**